monai/networks/nets/hovernet.py

```diff
@@ -0,0 +1,464 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from collections import OrderedDict
+from typing import Callable, Sequence, Type, Union, List
+
+import torch
+import torch.nn as nn
+from torch.hub import load_state_dict_from_url
+
+from monai.networks.layers.factories import Conv, Dropout, Pool
+from monai.networks.layers.utils import get_act_layer, get_norm_layer
+from monai.networks.blocks import UpSample
+from monai.utils.module import look_up_option
+from monai.utils import InterpolateMode, UpsampleMode
+#from monai.networks.blocks.localnet_block import ResidualBlock
+
+__all__ = [
+   "hovernet",
+    "Hovernet",
+    "HoVernet",
+    "HoVerNet",
+    "HoverNet",
+]

```

Comment:
If you want to have all the variation, you should add the aliases at the end of the file too. Like here:
https://github.com/Project-MONAI/MONAI/blob/29c62198d24b38688c6dd656bac3ea3a5c96af5b/monai/transforms/croppad/dictionary.py#L1526